lua/sig-window-nvim.lua

```diff
@@ -69,18 +78,22 @@ local function close_signature_window(bufnr)
   end
 end
 
-local function show_signature_window(label, active_ix_start, active_ix_end, config)
+local function show_signature_window(label, active_ix_start, active_ix_end, config, other_labels)
   local bufnr = vim.api.nvim_get_current_buf()
   local w_bufnr = vim.api.nvim_create_buf(false, true)
-  vim.api.nvim_buf_set_lines(w_bufnr, 0, -1, true, {label})
+
+  table.insert(other_labels, 1, label)  -- put activeSignature label the first line
+  vim.api.nvim_buf_set_lines(w_bufnr, 0, -2, true, other_labels)
   highlight_text(w_bufnr, active_ix_start, active_ix_end, config.hl_group)
 
-  local width, height = calc_window_dimensions(label, config.max_width, config.max_height)
+  local lines = other_labels
+  local width, height = calc_window_dimensions(lines, config.max_width, config.max_height)
   local winnr = vim.api.nvim_open_win(w_bufnr, false, window_config(label, config, width, height))
```

Comment:
I think window_config should be updated to take other_labels (without the active label) as the last argument, but that doesn't need to be done in this PR. I will take that change on and update the documentation.